### ISSUE TITLE
Commands to the terminal are undocumented

### ISSUE THREAD

**Issue by marcedelmann** [ElkeF/tc_commands#27]
This issue affects the file:
terminal_basics.txt

Are contributions to interactions with the terminal considered helpful in this document?

Currently, a subset of cureutils is introduced here with minimal descriptions of ther usage and use cases. I propose an expansion of this document to basic commands issued to the terminal itself, starting with:

`CTRL` + `L` or
`⌃` + `L` on Apple
Holding the control (Ctrl) key while pressing the letter `L` key will clear the screen of the current terminal.

**Comment by ElkeF** [comment]
These are indeed interesting features.

But
a) the command clear does what you describe
b) ⌘ + L on Apple does not do, what you describe on my machine. CTRL + L however does. Are you sure about this?

I am thinking about two options:
i) just add "clear" to the file
ii) add a new file with additional helpful shortcuts, CTRL + d and CTRL + c could go there as well

What do you think?

**Comment by marcedelmann** [comment]
> But 
> a) the command clear does what you describe
> b) ⌘ + L on Apple does not do, what you describe on my machine. CTRL + L however does. Are you sure about this?

You are correct, I've had the keys mixed up and edited my original suggestion accordingly: Use `⌃`+ `L` on Apple devices.

> I am thinking about two options:
> i) just add "clear" to the file
> ii) add a new file with additional helpful shortcuts, CTRL + d and CTRL + c could go there as well
> 
> What do you think?

Agreed. The `clear` command follows the style of the document and should be added there.
With your additions a new file would be reasonble, indeed.

**Comment by ElkeF** [comment]
Are you going to implement it?
